File: Client/Core/Models/TaskModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager.Client.Core.Models
{
    public class TaskModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime? DueDate { get; set; }
        public string CreatedByName { get; set; }
        public string AssignedToName { get; set; }
        public string Status { get; set; }
    }
}

File: Client/App.UI/TaskManagerApp/src/app/data/models/task.model.ts
export interface TaskModel{
    Id: number;
    Title:string;
    Description:string;
    DueDate:Date;
    AssignedToName:string;
    CreatedByName:string ;
    DateCreated:Date;
    Status:string;
}
export interface AddTaskModel{
    Title:string;
    Description:string;
    Duedate:Date;
    AssigneeId:number;
    CreatedUserId:number ;
    DateCreated:Date;
    StatusId:number;
}